### PR TITLE
create GitHub page for Configuration.md

### PR DESCRIPTION
**Pull request by rchaser53**
related: https://github.com/rust-lang/rustfmt/issues/3455

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                  configurationAboutHtml: '',
+                  searchCondition: '',
+                  configurationDescriptions,
+                  shouldStable: false
+                }
+              },
+              computed: {
+                outputHtml() {
+                  const ast = this.configurationDescriptions
+                                  .filter(({ head, text, stable }) => {
```
Comment (scampi) [inline docs/index.html]: the `head` variable is not used.

Comment (rchaser53) [reply docs/index.html]: Thank you for the review. I include head for search condition.

Comment (scampi) [reply docs/index.html]: Thanks

**Comment by rchaser53** [comment]
fix css for android chrome

**Comment by scampi** [comment]
@topecongiro what do you think about the page ? I think it looks ok and is useful

**Pull request merged**